cdap-watchdog/src/main/java/co/cask/cdap/metrics/data/AggregatesTable.java

```diff
@@ -59,76 +57,47 @@
   }
 
   /**
-   * Atomically compare a single metric entry with an expected value, and if it matches, replace it with a new value.
-   *
-   * @param context the context of the metric.
-   * @param metric the name of the metric.
-   * @param runId the runid of the metric.
-   * @param tag the tag of the metric.  If null, the empty tag is used.
-   * @param oldValue the expected value of the column. If null, this means that the column must not exist.
-   * @param newValue the new value of the column. If null, the effect to delete the column if the comparison succeeds.
-   * @return whether the write happened, that is, whether the existing value of the column matched the expected value.
-   */
-  public boolean swap(String context, String metric, String runId, String tag, Long oldValue, Long newValue)
```

Comment:
True, unrelated but it was affected by the refactor. Better to remove than to refactor.